source/process/bug-fix-release.md

```diff
@@ -168,19 +168,20 @@ Review the [Release Features & Bugs Quality Gate Guidelines](https://docs.google
     - Find QA or other teammates to help finish unfinished tests if needed
     - End of day or next morning: Verify all release tests are finished, bring any concerns to Triage / Release meeting
     - Go through all tabs of testing spreadsheet and verify all comments and questions have been filed in Jira as needed
-    - Verify all Jira tickets other than newly-filed bugs have been tested, verified, and closed
-    - As bug fixes are merged and RCs are cut, verify fixes on new RCs, and post in Release Channel after testing
-    - As RCs are cut, update `selenium.test.mattermost.com` to latest RC
     
 ### I. (T-minus 5 working days) Code Freeze
-Day after which only **S1 regressions** should be fixed (crashes, security vulnerabilities) and no other code changes should be comitted. Exceptions can be made by the Release Manager during triage on a case-by-case basis. Review the [Release Features & Bugs Quality Gate Guidelines](https://docs.google.com/document/d/1QxB_A1qkEJBKAvQpRa7JiSQLZhwg6HAEajNRNa7ldGg/edit)
+Day after which only **S1 regressions** should be fixed (crashes, security vulnerabilities) and no other code changes should be committed. Exceptions can be made by the Release Manager during triage on a case-by-case basis. Review the [Bug Severity Guidelines](/process/bug-severity-guidelines).
```

Comment:
Hm, I realized my suggestion for the link should probably be `(https://docs.mattermost.com/process/bug-severity-guidelines)`. I missed out the beginning of the link. Does that seem right? Sorry about that.